src/components/footer/_macro.njk

```diff
@@ -8,20 +8,38 @@
+            </svg>
+
         {% else %}
-            {% set poweredByAlt = 'Office for National Statistics' %}
-        {% endif %}
-    {% endif %}
 
-    {% set poweredByLogo %}
-        <img src="{{ params.assetsURL if params.assetsURL }}/img/{% if params.poweredBy is defined and params.poweredBy and params.poweredBy.logo is defined and params.poweredBy.logo %}{{ params.poweredBy.logo }}{% else %}{{'ons-logo-black-' + lang }}{% endif %}.svg" alt="{{ poweredByAlt | default('Office for National Statistics') }}">
+            {% if params.lang == 'cy' %}
+                <svg xmlns="http://www.w3.org/2000/svg" width="189px" height="19px" viewBox="0 0 189 19" focusable="false">
```

Comment:
Alt text is no longer able to be set. We should keep the ability to set the alt text with a default as backup

Reply:
Why alt text @rmccar they are now SVG’s. We know what SVG’s we are using now.

Reply:
Ok, I am updating to match what we have in the header.